server/src/modules/rules/rules.service.ts

```diff
@@ -383,6 +386,7 @@ export class RulesService {
           params.isActive !== undefined ? params.isActive : true,
           params.dataType !== undefined ? params.dataType : undefined,
           group.id,
+          params.notifications,
```

Comment:
This also needs adding in setRules when calling createOrUpdateGroup, I can't save new collections at the moment due to the error below, but it also means it won't save any notifications I've selected as well.

The notifications param on createOrUpdateGroup is optional but .map() is called on it later. This is throwing causing groupId to be null in setRules.